test/plugins/leap.test.ts

```diff
@@ -0,0 +1,144 @@
+import { Configuration } from '../testConfiguration';
+import { newTest } from '../testSimplifier';
+import { cleanUpWorkspace, setupWorkspace } from '../testUtils';
+
+function leapCommand(key: 's' | 'S', searchStrings: string[], jumpKeys: string[] = []) {
+  return [key, ...searchStrings, ...jumpKeys].join('');
+}
+
+suite('leap plugin', () => {
```

Comment:
I'd like to see some more test cases - across lines for instance

Reply:
Okay, I'll add a more comprehensive test case.